fe/src/main/java/org/apache/doris/load/loadv2/LoadManager.java

```diff
@@ -294,6 +294,10 @@ public void submitJobs() {
                 loadJob -> loadJob.state == JobState.PENDING).collect(Collectors.toList()));
     }
 
+    Map<Long, LoadJob> getIdToLoadJobs() {
```

Comment:
Why remove the qualifier of the method and `void addLoadJob(LoadJob loadJob)`

Reply:
Because `LoadManagerTest` need access  these method.

Reply:
You can use the invoke function to access these method.

Reply:
OK, but I think it is better to add an explicit qualifier~

Reply:
> You can use the invoke function to access these method.
I don't think use reflection is necessary in here.

Reply:
> OK, but I think it is better to add an explicit qualifier~

The accessible level for Java access modifiers is :
public > protected >  default(package) > private,

and we should keep accessible level as small as possible.

So I think use default(package) access modifier is reasonable here.

Reply:
> > You can use the invoke function to access these method.
> 
> I don't think use reflection is necessary in here.

The function of Deencapsulation.invoke() specifically designed to test private method without change private to default.
So maybe Deencapsulation.invoke() is better? It can keep the smaller accessible level for `getIdToLoadJobs`

Reply:
@EmmyMiao87 I got it!
I misunderstood the `invoke` word before.
Thanks you!